components/basic/ProfileImageInput.js

```diff
@@ -77,10 +77,7 @@ function ProfileImageInput({ image, setImage, className }) {
         />
         {/* image input with label */}
         <div>
-          <label
-            htmlFor="mentorProfile"
-            className="mentorFormButton theme-button-color !tw-font-medium"
-          >
```

Comment:
what this class is causing? I wonder how this cause issue not to get the updated profile imate. As this component is being used everywhere in grabtern and I recently. checked it was working at my end

Reply:
` <label htmlFor="mentorProfile">
            {!image ? "Upload Image" : "Change Image"}
          </label>`  this is a label , but the styles are used for buttons and  color is purple , even without !tw-font-medium it still remains in the bold font 
this is how it looks now , 
![image](https://github.com/user-attachments/assets/1396650b-1713-4fd4-9136-c6f1354d9c13)
this how it looked before
![image](https://github.com/user-attachments/assets/632536ec-e545-4360-975d-4ebf4d27b38c)

Reply:
oh that was a condition based, when you upload it changes to change Image. ( before is fine, and I guess it is want is required) let's keep this for tomorrow meeting disscussion.

Reply:
Hi anmol , i have just push a new commit , i finally found the issue why it is not showing in Profile.js coz the htmlfor and id of the lable and ip are begin duplicated (the id "mentorProfile" is also used some where else in the page) i just change the id to mentorProfileOne , now it will work fine in any page also i restored the org styles